### ISSUE TITLE
failed to execute golangci-lint: cannot parse result: EOF

### ISSUE THREAD

**Issue by sheremetat** [Mushus/golangci-linter#3]
When I try to set parameter `config` with my custom golangci-lint config file, I've got following error:

`failed to execute golangci-lint: cannot parse result: EOF`

Without this parameter (with default  golangci-lint config file) everything works fine

**Comment by Mushus** [comment]
duplicate #2

**Comment by sheremetat** [comment]
Hm, I'm not sure. I have already tried to use `--config .golangci.yml` instead of `.golangci.yml` as `config` parameter value and this error has appeared in logs. 

BTW, I will try to test it again just after new version 1.1.1 will be available on the marketplace.

**Comment by sheremetat** [comment]
Hi @Mushus, I just tested a new version. Everything works fine. Thanks a lot! Also, the readme file should be updated.

**Comment by maksim77** [comment]
Unfortunately, in my case, the latest version did not solve the problem.
https://github.com/maksim77/tinkoff_exporter/commit/d5fec5dfbb064067568b355b1d3173debf4b7873/checks?check_suite_id=401681173

**Comment by Mushus** [comment]
GolangCI-Lint may not return expected JSON format.

**Comment by maksim77** [comment]
> GolangCI-Lint may not return expected JSON format.

Maybe we must reopen this ticket?

**Comment by Mushus** [comment]
This may be because the `actions/setup-go` has added environment variables `GOROOT`

not working ( after actions/setup-go )
```
/usr/bin/docker run --name ee613286f23469c492988505b506ae81e40_3d3e5c --label 671ee6 --workdir /github/workspace --rm -e GOROOT -e INPUT_CONFIG -e INPUT_BASEPATH -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/tinkoff_exporter/tinkoff_exporter":"/github/workspace" 671ee6:13286f23469c492988505b506ae81e40
```
working
```
/usr/bin/docker run --name ee68613848f6ef24ff0aef2199d910ab022_186ae2 --label 671ee6 --workdir /github/workspace --rm -e INPUT_CONFIG -e INPUT_BASEPATH -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/tinkoff_exporter/tinkoff_exporter":"/github/workspace" 671ee6:8613848f6ef24ff0aef2199d910ab022
```

**Comment by Mushus** [comment]
Released a new version `1.1.2` fixes this problem
